driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpChildChannelSource.java

```diff
@@ -147,7 +148,10 @@ protected void httpMessageReceived(ChannelHandlerContext ctx, MessageEvent e, Ht
             return;
         }
 
-        Entry<URI, HttpServerChannel> httpBinding = httpBindings.floorEntry(httpLocation);
+        ChannelAddress transportCandidate = Channels.localAddress(ctx.getChannel());
```

Comment:
Are you suggesting that we store the resolved address in bindings map ? Somewhere we need to associate channel's local address with bind address. I have some changes to populate that but seems like a hack. We can iterate in the next review